### PR TITLE
build: Remove redundant build files

### PR DESCRIPTION
**Pull request by kumaraditya303**
Remove redundant build files .

### CODE REVIEW

File: .github/workflows/ci.yml [hunk 1]
```diff
@@ -47,28 +47,9 @@ jobs:
     strategy:
       fail-fast: false
       matrix:
-        os: [ubuntu-latest, windows-latest, macos-latest]
-        python-version: [3.7, 3.8]
+        os: [ubuntu-latest, windows-latest, macos-latest, macos-11.0]
+        python-version: [3.7, 3.8, 3.9]
         browser: [chromium, firefox, webkit]
-        include:
```
Comment (mxschmitt) [inline .github/workflows/ci.yml:53]: This was intended and by your change it increases the jobs from 26 to 38 which is I think not needed. It's already checked with Python 3.9.

File: scripts/update_versions.py [hunk 2]
```diff
@@ -2,20 +2,27 @@
...
-            text,
-        )
-        browser.close()
 
-    w = open("README.md", "w")
-    w.write(text)
-    w.close()
+def main() -> None:
+    with sync_playwright() as p:
+        with open("README.md", "r") as r:
```
Comment (mxschmitt) [inline scripts/update_versions.py:8]: using pathlib with read_text and write_text is probably cleaner then it's less nested and redundated code.

Comment (kumaraditya303) [reply scripts/update_versions.py:8]: I was just about to do it 🚀

File: .github/workflows/ci.yml (hunk 1 above)
Comment (kumaraditya303) [reply .github/workflows/ci.yml:53]: @mxschmitt I changed it as I assumed as this project is open source so it must be on a free tier. Anyway if you think so I will change it

**Comment by kumaraditya303** [comment]
@mxschmitt I think https://github.com/microsoft/playwright-python/blob/master/docs/development.md should be merged in https://github.com/microsoft/playwright-python/blob/master/CONTRIBUTING.md

File: scripts/test_to_python.js [hunk 3]
```diff
@@ -1,164 +0,0 @@
-#!/usr/bin/env node
```
Comment (pavelfeldman) [inline scripts/test_to_python.js:1]: I'm still using this one for the new tests

Comment (kumaraditya303) [reply scripts/test_to_python.js]: @pavelfeldman  Added it again

**Comment by kumaraditya303** [comment]
@pavelfeldman PR is rebased on master, you can review now

File: scripts/update_versions.py [hunk 4]
```diff
@@ -1,21 +1,26 @@
...
-            text,
-        )
-        browser.close()
 
-    w = open("README.md", "w")
-    w.write(text)
-    w.close()
+def main() -> None:
+    with sync_playwright() as p:
+        text = Path("README.md").read_text(encoding="utf-8")
```
Comment (mxschmitt) [inline scripts/update_versions.py:9]: store Path("README.md") in a temporary variable, it's redundant.

**Review by mxschmitt — APPROVED** [review]

File: .github/workflows/publish.yml [hunk 5]
```diff
@@ -60,8 +56,9 @@ jobs:
         # GITHUB_REF has a form of `refs/tags/v1.3.0`.
         # TAG_NAME would be `v1.3.0`
         TAG_NAME=${GITHUB_REF#refs/tags/}
-        ./scripts/tag_image_and_push.sh playwright-python:localbuild playwright.azurecr.io/public/playwright-python:latest
-        ./scripts/tag_image_and_push.sh playwright-python:localbuild playwright.azurecr.io/public/playwright-python:${TAG_NAME}
-
-        ./scripts/tag_image_and_push.sh playwright-python:localbuild playwright.azurecr.io/public/playwright-python:focal
-        ./scripts/tag_image_and_push.sh playwright-python:localbuild playwright.azurecr.io/public/playwright-python:${TAG_NAME}-focal
+        docker tag playwright-python:localbuild playwright.azurecr.io/public/playwright-python:latest
```
Comment (kumaraditya303) [inline .github/workflows/publish.yml:59]: `docker push --all-tags` is much cleaner than using an custom script to tag and build every image

Comment (aslushnikov) [reply .github/workflows/publish.yml:59]: We tried it awhile before in our main repo `microsoft/playwright` and it didn't work. Thus the script 🤷‍♂️

I wouldn't touch this for now unless there's a strong urge.

Comment (kumaraditya303) [reply .github/workflows/publish.yml:59]: @aslushnikov sure it would revert it

**Comment by kumaraditya303** [comment]
@pavelfeldman PR rebased and ready to be merged :rocket:

**Comment by kumaraditya303** [comment]
@mxschmitt Can you review and get this merged ?

**Comment by kumaraditya303** [comment]
Thanks @mxschmitt

**Pull request merged**